src/bubble.ts

```diff
@@ -356,6 +379,19 @@ export default class Bubble {
         this.setBubbleSpec(this.spec);
       });
     }
+    // Now, look for a child whose joiner should be our center, and adjust that.
+    const child = Comical.findChild(this);
+    if (child) {
+      child.adjustJoiners(contentCenter);
+    }
+  }
+
+  private adjustJoiners(newTip: Point): void {
+    this.tails.forEach((tail: Tail) => {
+      if (tail.spec.joiner && tail.adjustTip(newTip)) {
+        this.persistBubbleSpec();
```

Comment:
Food for thought:
Which is better here? persistBubbleSpec() or persistBubbleSpecWithoutMonitoring?